conf/app.routes

```diff
@@ -127,8 +127,8 @@ GET         /events/:venue/:eventType                                    control
 PUT         /events/:eventId/allocate                                    controllers.EventsController.allocateAssessor(eventId: String)
 
 # Day aggregate events
-GET         /day-aggregate-events/skills/:skillTypes                     controllers.DayAggregateEventController.findBySkillTypes(skillTypes: String)
-GET         /day-aggregate-events/location/:location/skills/:skillTypes  controllers.DayAggregateEventController.findBySkillTypesAndLocation(skillTypes: String, location: String)
+GET         /day-aggregate-events                                        controllers.DayAggregateEventController.findBySkillTypes(skills: List[model.persisted.eventschedules.SkillType.SkillType])
```

Comment:
This is the right package for the enum - I think we need to look at moving to AssessorSkill case class in some way as we've introduced that last week.